account_product_move/models/account_product_move.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import _, api, fields, models
+from odoo.tools import UserError
+
+
+class AccountProductMove(models.Model):
+    _name = "account.product.move"
+    _description = "Template for additional journal entries/items"
+
+    _sql_constraints = [
+        (
+            "product_tmpl_id",
+            "UNIQUE (product_tmpl_id)",
+            _("Template for this product already exists"),
+        ),
+    ]
+
+    name = fields.Char(required=True, copy=False, default="New")
+    product_tmpl_id = fields.Many2one(
+        comodel_name="product.template",
+        string="Product",
+        help="Journal items will be created for this product",
+    )
+    journal_item_ids = fields.One2many(
+        comodel_name="account.product.move.line",
+        inverse_name="product_move_id",
+        copy=True,
+        string="Extra Journal Items",
+        help="Journal items to be added in new journal entry",
+    )
```

Comment:
Am I correct in assuming that an `account.product.move` record would end up being create as a move (Journal Entry) and the lines as underlying move lines (Journal Item)?